### ISSUE TITLE
Installation - AUR package link

### ISSUE THREAD

**Issue by hendrimat** [johnwmillr/LyricsGenius#89]
I created a package of LyricsGenius for Arch Linux and published it to AUR.
Maybe you could put Arch Linux installation instructions under "Installation" like this:

Install the AUR package for Arch Linux manually:
```
curl -L -O https://aur.archlinux.org/cgit/aur.git/snapshot/python-lyricsgenius.tar.gz
tar -xvf python-lyricsgenius.tar.gz
cd python-lyricsgenius
makepkg -si
```

**Comment by johnwmillr** [comment]
Hi there,

I'm unfamiliar with AUR packages, could you explain their use? Why isn't the usual `pip install` a valid option on Linux?

John

**Comment by hendrimat** [comment]
Arch Linux is a Linux distribution, like Ubuntu, and AUR is a community-driven repository, that provides packages which are not in the official repositories. The problem with Python packages, that are installed with pip, is that they may (and usually do) conflict with the system packages. To avoid that, many Python packages are ported over to the AUR or official repos.

The reason, why I created the package is that I have a script, which displays lyrics to the currently playing song, and I needed to install your excellent LyricsGenius package to use it system-wide.

Also I think you probably don't need to add the full instructions, maybe just a sentence saying that it is available to Arch users on the AUR.

Hendrik
